Assets/Scripts/ProjController.cs

```diff
@@ -0,0 +1,66 @@
+using UnityEngine;
+
+public class ProjController : MonoBehaviour
+{
+    [SerializeField]
+    private int projDamage = 100;
+    [SerializeField]
+    private float projMoveSpeed = 7f;
+    [SerializeField]
+    private bool wantDeviation = true;
+    [SerializeField]
+    private float projDeviation = 1f;
+
+    public string target = "null"; // Needs to be set on creation by creating object, "Player" to damage players, "Enemy" to damage enemies
+    public Transform targetPos; // Needs to be set on creation by creating object
+
+    private Rigidbody2D rb;
+    private Vector2 moveDirect; // Movement Direction
+
+    void Start()
+    {
+        rb = GetComponent<Rigidbody2D>();
+        moveDirect = (targetPos.position - transform.position).normalized * projMoveSpeed;
+
+        if (wantDeviation)
+        {
+            moveDirect.x = Random.Range(moveDirect.x - projDeviation, moveDirect.x + projDeviation);
```

Comment:
I would prefer calculating a deviation vector and then adding that to `moveDirect`, but this way is also fine with me.

Reply:
Is there a reason to do that, beyond personal preference?

Reply:
I think it simplifies the code a bit. It also makes it easier to extract deviation calculation into a method, but that probably isn't something we'll need to do. You can leave it as is.